WORKSPACE

```diff
@@ -139,6 +139,14 @@ git_repository(
     shallow_since = "1647554845 -0700",
 )
 
+# A custom fork of Android-MarqueeView that is updated with latest dependencies compatible with Oppia and also
+# min target SDK version set to be compatible with Oppia.
+git_repository(
+    name = "marqueeview",
```

Comment:
Per earlier comment, reminder to add the shallow_since (you should be able to find the recommended line to add from the build logs run in CI).

Reply:
Noted.